apps/cowswap-frontend/src/modules/permit/utils/buildFakeSigner.ts

```diff
@@ -0,0 +1,8 @@
+import { Wallet } from '@ethersproject/wallet'
+
+/**
+ * Builds a fake EthersJS Wallet signer to use with EIP2612 Permit
+ */
+export function buildFakeSigner(): Wallet {
+  return Wallet.createRandom()
```

Comment:
I would prefer to use a PK, to make it more deterministic. 
Note that the address might impact the gas in some situations, so if we need to adjust some "gas buffer" for some estimations, we better experiment with a determisitic address